README.md

```diff
@@ -74,6 +74,7 @@ We also have a list of <a href="UNMAINTAINED.md">unmaintained projects</a>. If y
 ### Native Editors
 
 * [JOSM](https://josm.openstreetmap.de) - Advanced extensible desktop editor. ([Source Code](https://josm.openstreetmap.de/browser) / [Wiki](https://wiki.openstreetmap.org/wiki/JOSM))
+* [Merkaartor](https://github.com/openstreetmap/merkaartor) - An opensource OSM editor, written in C++ and Qt
```

Comment:
Hey @sikmir,

Thanks for your contribution and sorry for not getting to it sooner!

In awesome-lists, new entries are always supposed to be added at the bottom instead of alphabetically, so if you do that for Merkaartor I'll be able to merge 😃